Tests/test_Seq_objs.py

```diff
@@ -19,10 +18,8 @@
 from Bio.Seq import Seq, UnknownSeq, MutableSeq, translate
 from Bio.Data.CodonTable import TranslationError, CodonTable
 
-if sys.version_info[0] < 3:
-    from string import maketrans
-else:
-    maketrans = str.maketrans
+
+maketrans = str.maketrans
```

Comment:
Maybe just call `str.maketrans` each time (if others agree)

Reply:
yeah, I guess this is the better way. I want @peterjc and @mdehoon to confirm this change.

Reply:
Either way seems fine by me.

Reply:
@mdehoon what do you suggest here?

Reply:
Call `str.maketrans` each time, as suggested by @chris-rands

Reply:
fixed this.